### PR TITLE
signup  page

### PR DESCRIPTION
**Pull request by prath-dev**
Screenshot from 2022-02-22 00-20-39

### CODE REVIEW

**Review by DevNirah7 — APPROVED** [review]
Looks clean. Good to commit!

File: Components/Css/signup.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.login-form {
+    margin-left: 35rem;
+}
```
Comment (anupkrbid) [inline Components/Css/signup.css:3]: you should use flexbox or grid to center the form, or at least use `margin: auto`.

File: Views/signup.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pdui.netlify.app/pdui.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
```
Comment (anupkrbid) [inline Views/signup.html:9]: add this link to your CSS file so you do not have to add it to every HTML file.

**Review by MonikaJhingan — APPROVED** [review]
Ui looks neat good to be merged.

File: Views/signup.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        </div>
+        <div>
+            <label for="">Password</label>
+            <input placeholder="******" class="input-box" type="password">
+            <div>
+                <input class="btn" type="button" value="Login">
+            </div>
+            <div class="checkbox">
+                <input type="checkbox">
+                <span>Remember Me</span>
```
Comment (anupkrbid) [inline Views/signup.html:42]: You should use a label for the checkbox instead of a span

**Review by anupkrbid — APPROVED** [review]
A few suggestions, but overall okay.

**Comment by prath-dev** [comment]
@MonikaJhingan @anupkrbid thansk for your suggestion  i will do changes in second version of my ecommerce..thx for your time guys..

**Pull request merged**